content/300-guides/200-deployment/110-deployment-guides/650-caveats-when-deploying-to-aws-platforms.mdx

```diff
@@ -32,6 +32,13 @@ unsafe-perm=true
 
 Enabling `unsafe-perm` forces _npm_ to run as _root_, avoiding the filesystem access problem, thereby allowing the `prisma generate` command in the `postinstall` hook to generate code into `node_modules`.
 
+### Error: @prisma/client did not initialize yet
+
+This error happens because AWS Elastic Beanstalk doesn't install `devDependencies`, which means that it doesn't pick up the Prisma CLI. To remedy this you can either:
+
+1. Add the `prisma` package to your `dependencies` instead of the `devDependencies`. (Making sure to run `npm install` afterwards to update the `package.lock.json`).
```

Comment:
```suggestion
1. Add the `prisma` CLI package to your `dependencies` instead of the `devDependencies`. (Making sure to run `npm install` afterward to update the `package-lock.json`).
```